manifests/kustomize/base/kustomization.yaml

```diff
@@ -7,6 +7,8 @@ bases:
 - pipeline
 - proxy
 - metadata
+- cache
+- cache-deployer
```

Comment:
I think we should leave these out, because caching feature won't be ready before next release. IMO, we should include caching in deployment by default only after it is feature complete and we are confident with it.
Before that, it should be deployed manually as an optional component (but it's fine programmatically deploy it in presubmit tests if you want test coverage).

I think we can focus on the most important features first, when the feature is ready, it isn't too late for us to discuss how we should handle deployment and upgrade solutions. (I found kubeflow is handling admission webhook bootstrap in a statefulset https://github.com/kubeflow/manifests/blob/master/admission-webhook/bootstrap/base/stateful-set.yaml#L20, that's probably a way to avoid problem with `Job`. (As I mentioned, we can dig into these details later, they are not core part of this feature. Just go with whatever works now.)

/cc @IronPan

Reply:
good point. +1